content/main.js

```diff
@@ -464,10 +464,21 @@ const inspectElement = async (element) => {
 }
 
 const inspectAllMediaElements = () => {
-  const elements = [
-    ...document.getElementsByTagName('video'),
-    ...document.getElementsByTagName('audio'),
-  ];
+  function getElementsRecursive(rootEl) {
+    // Get all media elements within this root element context
```

Comment:
```suggestion
    if (!rootEl) return [];
    
    // Get all media elements within this root element context
```

I noticed that I get errors on some pages where `frame.contentDocument` is undefined due to permission restrictions so I'd suggest adding this small check.

Reply:
Good catch :) I've committed your suggested edits!